### ISSUE TITLE
handleSubmit is not a function (using ValidationObserver)

### ISSUE THREAD

**Issue by yahmez00** [lewyuburi/nuxt-validate#25]
I've set up my form in a similar manner to this: https://logaretm.github.io/vee-validate/guide/forms.html#validate-before-submit

However when I use the `ValidationObserver` the `invalid` and `handleSubmit` slots don't seem to work?

Everything else seems fine so I'm not sure if I'm missing something. Any help would be great, thanks!

**Comment by simplenotezy** [comment]
I get the same error, with the "handleSubmit" error. I have opened  an issue here: https://github.com/logaretm/vee-validate/issues/2662

**Comment by nlxdodge** [comment]
Yeah so that was only available from version 3.2 according to the link above. So this project is out of date, as it uses version 3.0.8.

**Comment by chapdel** [comment]
Actualy, nuxt-validate use vee-validate v2
check with ```bash npm list```
